drivers/aufs/aufs.go

```diff
@@ -118,14 +118,10 @@ func Init(root string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		locker:    locker.New(),
 	}
 
-	rootUID, rootGID, err := idtools.GetRootUIDGID(uidMaps, gidMaps)
-	if err != nil {
-		return nil, err
-	}
 	// Create the root aufs driver dir and return
 	// if it already exists
 	// If not populate the dir structure
-	if err := idtools.MkdirAllAs(root, 0700, rootUID, rootGID); err != nil {
+	if err := os.MkdirAll(root, 0711); err != nil {
```

Comment:
I take it that this is a directory which will just be mounted on?  SO you just need search perms in the UserNamespace?

Reply:
It's the top level for the driver (`/var/lib/containers/storage/aufs`), while the filesystems that get mounted are at `mnt/$layerid` under it.  We've seen permissions-related problems preventing us from launching containers using user namespaces when they were using root directories which had some parent directory wasn't at least searchable by the UID/GID that we were mapping to 0/0 for the container.

Reply:
Ok. SGTM